.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseCJKNumberParser.cs

```diff
@@ -477,7 +477,6 @@ private double GetIntValue(string intStr)
             var isDozen = false;
             var isPair = false;
             char roundNumberZero = '零';
```

Comment:
This is still Chinese-specific, right? So it should move to a declaration in the right YAML and referenced in the right culture config.